subprojects/shipkit/src/test/groovy/org/shipkit/gradle/ShipkitGradlePluginIntegTest.groovy

```diff
@@ -44,7 +44,6 @@ class ShipkitGradlePluginIntegTest extends GradleSpecification {
 :buildArchives
 :gitTag
 :identifyGitBranch
-:identifyGitOrigin
```

Comment:
So this task is not a part of performRelease now? Is it only used in initShipkitFileTask?

Reply:
It is used in:
 - init scenarios
 - upgradeDownstream/versionUpgrade scenarios

We cannot really use it for other scenarios, e.g. to get rid of 'repository name' setting in shipkit file. I really tried :)

Example use case: if I fork mockito repository, clone it to my local, prepare a PR and want to preview the release notes, I need the release notes generation logic to use "mockito/mockito" repo, and not "szczepiq/mockito" repo. For now, we will keep the repository name setting in the shipkit file.